SECURITY_NOTES.md

```
@@ -0,0 +1,3 @@
+# 9 upstream vulns in openclaw@2026.4.11 (Lark/Discord deps)
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)